pom.xml

```diff
@@ -65,8 +65,22 @@
 			<artifactId>mapstruct</artifactId>
 			<version>1.5.5.Final</version>
 		</dependency>
+		<dependency>
+			<groupId>org.liquibase</groupId>
+			<artifactId>liquibase-core</artifactId>
+			<version>4.24.0</version>

```

Comment:
no need to specify the version - it is taken from `spring-boot-starter-parent`

Reply:
it doesn't work without them